modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/metadata/MetadataMediator.java

```diff
@@ -183,10 +190,10 @@ public MetadataMediator(RuntimeComponentModel componentModel)
      */
     private Optional<TypeMetadataDescriptor> getContentMetadataDescriptor()
     {
-        return contentParameter.isPresent() ? Optional.of(MetadataDescriptorBuilder.typeDescriptor(contentParameter.get().getName())
-                                                         .withType(contentParameter.get().getType())
-                                                         .build())
-                                            : Optional.empty();
+        return contentParameter != null ? Optional.of(MetadataDescriptorBuilder.typeDescriptor(contentParameter.get().getName())
```

Comment:
why? contentParameter should never be `null`, we have to make sure of that. 
Besides, if this is not null, but it is absent, this throws an exception on `contentParameter.get()`